pkgs/games/minosoft/default.nix

```
@@ -0,0 +1,101 @@
+{ lib, stdenv
+, makeWrapper
+, fetchFromGitLab
+, gradle
+, perl
+, jre
+, libpulseaudio
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+let
+  pname = "minosoft";
+  version = "7ba4569"; # there are no tags yet, so lets version this using tags for now
+
+  src = fetchFromGitLab {
+    domain = "gitlab.bixilon.de";
+    owner = "bixilon";
+    repo = "minosoft";
+    rev = "7ba45697e3586e0b6de231ed5db6d3e34cb37d82";
+    hash = "sha256-sro49mh6c20HN1ZAHp5Ukzmn5IcniuLTa0w43S8ZVNw=";
+    leaveDotGit = true; # needed by gradle scripts
+  };
+
+  # build jar into fixed-output derivation (it is fetching JARs to include from the internet anyways, so it doesnt matter if its not that conform to our ideals)
+  deps = stdenv.mkDerivation {
+    pname = "${pname}-deps";
```

Do not template `pname`:


```suggestion
    pname = "minosoft-deps";
```